src/components/StoreList/StoreList.js

```diff
@@ -1,13 +1,44 @@
 import React from 'react';
 import ButtonList from '../ButtonList/ButtonList';
 import getStore from '../../utils/honestyStore.js';
-import sendSlackMessage from '../../utils/slack';
 
 export default class StoreList extends React.Component {
 	state = {
 		storeList: []
 	};
 
+	usernameToID = username => {
+		console.log(this.props.users);
+		let users = this.props.users;
+		let i = users.length;
+		while (i--) {
+			if (
+				users[i].name === username ||
+				users[i].profile.real_name === username
+			) {
+				return users[i].id;
+			}
+		}
+		return null;
+	};
+
+	sendSlackMessage = async (username, itemName, storeCode) => {
+		let id = this.usernameToID(username);
+		if (!id) {
+			alert('Error: Slack user not found');
```

Comment:
Usually we try and avoid alert, it hijacks the browser and isn't a great user experience. Could this maybe be shown in the UI somehow instead?

(Same for other alert usages)